src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.bar {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid rgba(100, 100, 100, 0.2);
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.big__bar {
+  margin: 20px 0;
+  background-size: 19px;
+  background-position: left 26px top 25px;
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small__bar {
+  background-size: 11px;
+  background-position: left 13px top 15px;
+  padding-left: 33px;
+  height: 42px;
```

Comment:
Maybe there is a way to create this without height property and give to elements the ability to define the height, what do you think about it ?